content/webapp/components/SearchResults/SearchResults.tsx

```diff
@@ -112,23 +110,6 @@ const SearchResults: FunctionComponent<Props> = ({
             xOfY={{ x: index + 1, y: items.length }}
           />
         )}
-        {item.type === 'article-schedule-items' && (
-          <CompactCard
-            title={item.title || ''}
-            partNumber={item.partNumber}
-            color={item.color}
-            primaryLabels={
-              /* We don't show a label on items that haven't been published yet, because
-               * we don't know whether they're a story, comic, etc.
-               * See https://github.com/wellcomecollection/wellcomecollection.org/pull/7568 */
-              []
-            }
-            secondaryLabels={[]}
-            description={`Available ${formatDate(item.publishDate)}`}
-            Image={<ImagePlaceholder color={item.color} />}
-            xOfY={{ x: index + 1, y: items.length }}
-          />
-        )}
```

Comment:
This is another casualty of the type checker – it tells me that a MultiContent can never be an article-schedule-items, which makes sense – we don't transform this type in `parseMultiContent`. More code to come out!